UPGRADING.md

```diff
@@ -1,6 +1,30 @@
 Upgrading Hashie
 ================
 
+### Upgrading to 3.2.2
+
+#### Testing if key defined
+
+Versions <= 3.2.1 being questioned don't return a boolean value as it's mentioned in README.md
```

Comment:
"being questioned" is strange, as above. Looks like you're questioning a version :) Try to rewrite this into something more reasonable maybe. No big deal.